lib/tasks/export.rake

```
@@ -0,0 +1,47 @@
+    # Create book directory if it doesn't exist
+    book_dir = Rails.root.join('book')
+    FileUtils.mkdir_p(book_dir)
+    FileUtils.touch(book_dir.join('.keep'))
+
+    # Delete any existing files in the book directory
+    Dir.glob(book_dir.join('*.md')).each { |file| File.delete(file) }
+
+    # Export each chapter
+    book.chapters.order(:number).each do |chapter|
+      # Format the filename with padded number and sanitized name
+      filename = format('%02d_%s.md', chapter.number, chapter.name.downcase.gsub(/[^0-9a-z]/i, '_'))
```

RuboCop (1.69.1)

[convention] Prefer annotated tokens (like `%<foo>s`) over unannotated tokens (like `%s`).

(Style/FormatStringToken)